File: db_manager/models/enums.go
package models

// create here your enums...

type AuthorType string

const (
	PageType AuthorType = "page"
	UserType AuthorType = "user"
)

type FriendRequestStatus string

const (
	Pending  FriendRequestStatus = "pending"
	Rejected FriendRequestStatus = "rejected"
	Accepted FriendRequestStatus = "accepted"
)

type ReactionType string

const (
	Like  ReactionType = "like"
	Love  ReactionType = "love"
	Haha  ReactionType = "haha"
	Wow   ReactionType = "wow"
	Sad   ReactionType = "sad"
	Angry ReactionType = "angry"
)

var ReactionTypeLst = []string{
	string(Like),
	string(Love),
	string(Haha),
	string(Wow),
	string(Sad),
	string(Angry),
}

File: nosql/models/enums.py
from enum import Enum


class AuthorType(str, Enum):
    Page: str = "page"
    User: str = "user"


class FriendRequestStatus(str, Enum):
    Pending: str = "pending"
    Rejected: str = "rejected"
    Accepted: str = "accepted"


class UserPrivilege(str, Enum):
    Mod: str = "mod"
    Admin: str = "admin"
    User: str = "user"
